### DISCUSSION TITLE
请问如何将本科生学位论文模板的参考文献格式更改为本学科期刊（AMS）格式

### DISCUSSION THREAD

**Discussion by FakeChan** [nju-lug/NJUThesis#222] [Q&A] [answered]
如题，需要将参考文献格式从GB格式更改为
![屏幕截图 2023-05-29 160932](https://github.com/nju-lug/NJUThesis/assets/81480054/9e7763b1-dd8d-43e1-9cc1-e452f729045c)
已经有了该格式的bst和cls文件

**Comment by atxy-blip** [comment, marked as answer]
模板提供了一个可以禁用全部  `biblatex` 设置的选项，关掉后可以完全按照 BibTeX 的方式设置参考文献样式。

```tex
\documentclass[biblatex=false]{njuthesis}
\usepackage{natbib}
\bibliographystyle{...}
\begin{document}
...
\end{document}
```

当然你也可以从找一个  `biblatex` 样式（比如 <https://github.com/konn/biblatex-math>），把 `.bbx` `.cbx` 复制到项目目录下然后设置 `bib/style` 选项。

**Reply by FakeChan** [reply]
![image](https://github.com/nju-lug/NJUThesis/assets/81480054/79eff4e3-df15-46c4-8a21-1391a1598ec0)
你好，在更改设置后引用会出现大量报错，检查语法并没有发现任何问题，可能是什么原因呢？

**Reply by FakeChan** [reply]
问题已解决，清理编译缓存即可
